src/components/SignIn.js

```diff
@@ -1,26 +1,34 @@
-import { Auth } from 'aws-amplify'
 import React,{useState} from 'react'
```

Comment:
Sign in and sign up form, lets use react-hooks-form. That will save you lots of hassle.